Modules/CodeTrek/Resources/views/edit.blade.php

```diff
@@ -0,0 +1,79 @@
+@extends('codetrek::layouts.master')
+@section('content')
+
+<div class="container" id="update_details">
+    <div  class="card-body">
+        <form action="{{route('codetrek.update', $applicant->id)}}" method="POST" id='update_form' enctype="multipart/form-data">
```

Comment:
id should be in camelCase

```suggestion
        <form action="{{route('codetrek.update', $applicant->id)}}" method="POST" id='updateForm' enctype="multipart/form-data">
```